### DISCUSSION TITLE
how to set up multiple ports on my server

### DISCUSSION THREAD

**Discussion by freelovehere** [postalserver/postal#3012] [Help with using Postal]
i would love to set up multiple ports on my server but whenever i put two or more ports in postal.yml, it ends up showing only 1. Does anyne have an idea on how to go about it. I know its possible as i have seen the example somewhere and i have attached the proof. Thank you
![Screenshot 2024-06-20 at 02-58-40 Sending E-Mail - Help - Free Trial - happinessismine01@gmail com - Kingmailer](https://github.com/postalserver/postal/assets/169105514/6af92eb1-048d-4d21-a2a3-c052235b794b)

**Comment by willpower232** [comment]
Postal only listens on a single port so if you're going to use multiple ports, you would either need multiple smtp containers or to use iptables to forward the traffic around.

It is also worth mentioning that Postal only supports explicit TLS via STARTTLS so it may be that you actually cannot use all the standard ports as other email servers may expect it to be TLS by default.

**Comment by charlesmudy** [comment]
@willpower232 is there a guide on how to setup multiple smtp containers?
